onyo/lib/onyo.py

```diff
@@ -95,6 +152,11 @@ def clear_caches(self, assets: bool = True, dirs: bool = True,
 
         By default all caches are cleared, and flags make it possible to specify
         which caches which should remain set.
+
+        If the repository is only modified via the API functions, the caches of
+        the `Repo` object are kept consistent. If an external process modifies
+        the repository the caches must be cleared manually to restore the
+        integrity of the caches.
```

Comment:
integrity -> sanity ?

Not sure I like that, but integrity feels wrong.

Reply:
I think I will choose instead "consistency". "sanity" sounds missleading, it means kind of nothing, because who knows whats sane and what not? :P 
A lot of invalidation would be sane to expect in certain situations, I would argue.